### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by guks**
1. added the SVG logo to assets folder
2. the logo links to the home page
3. a tag on the left links to https://ooloo.io
4. router Link on the right links to /terms 
5. checked with the pixel perfect chrome extension
 screenshot_1

### CODE REVIEW

File: src/components/Footer/Footer.styles.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import { ReactComponent as LogoSVG } from '../../assets/images/sign.svg';
+
+export const FooterWrapper = styled.footer`
+  height: 100px;
+  display: flex;
+  justify-content: space-between;
+  margin-right: 251px;
+  margin-left: 251px;
+`;
+export const FooterLinkContainer = styled.span`
```
Comment (jkettmann) [inline src/components/Footer/Footer.styles.js:12]: newline missing

File: src/components/Footer/Footer.styles.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+
+import { ReactComponent as LogoSVG } from '../../assets/images/sign.svg';
```
Comment (jkettmann) [inline src/components/Footer/Footer.styles.js]: This is imported as base64 encoded string I guess. So I'd move the `sign.svg` inside the `src/components/Footer` directory. The logo is also not reused anywhere. But it's your call

**Review by jkettmann — CHANGES_REQUESTED** [review]
Nice job. Pixel-perfect! I left two minor comments

One more thing I'd like to see is that the footer is at the bottom of the window even if there is no content. You can do that by wrapping the `Switch` component in App inside a container with `min-height: calc(100vh - headerHeight - footerHeight)`

File: src/components/Footer/Footer.styles.js (hunk 2 above)
Comment (guks) [reply src/components/Footer/Footer.styles.js]: Agreed and changed

Comment (jkettmann) [reply src/components/Footer/Footer.styles.js]: I was actually saying that I would move sign.svg from `src/assets/images/sign.svg` to `src/components/Footer/sign.svg` and use it the way you did before

```
import { ReactComponent as LogoSVG } from './sign.svg'; 

export const Logo = styled(LogoSVG)`...`
```

But this is fine for me as well. I'll leave it up to you to change this or not

**Review by jkettmann — DISMISSED** [review]

File: src/components/Footer/Footer.styles.js (hunk 2 above)
Comment (guks) [reply src/components/Footer/Footer.styles.js]: Oh ok, I moved the images and deleted assets folder for now

**Review by jkettmann — APPROVED** [review]

**Pull request merged**